### DISCUSSION TITLE
Interesting example outside RGI and other peripheral glacier datasets in Antarctica

### DISCUSSION THREAD

**Discussion by B-Miles** [IACS-cryo/Delineation-WG#47] [Antarctica]
Been chatting to Bea, and thought i would share an example of an interesting region - Victoria Land, East Antarctica. For those familiar with Antarctica the larger glacier just cut of the image on the right hand side is Rennick Glacier that does penetrate into the inner main ice sheet. In this region virtually no glaciers are included in the RGI or other peripheral glacier datasets (see shading on figure). But i reckon there is decent argument to suggest that many of them are better placed as peripheral. The mountain range, at least visually appears to do a good job in separating these glaciers from the main ice sheet. Secondly, most of them are so narrow that I guess they are not included in current ice sheet projections (?) and would maybe be better suited to other modelling communities? 

![VictoriaLand-01](https://github.com/IACS-cryo/Delineation-WG/assets/154226002/c19ef5f3-f21a-4fff-bf8b-6eb18b82f54e)

**Comment by bearecinos** [comment]
@mankoff @fmaussion @MathieuMorlighem 
I downloaded a while ago the initial surface model velocity from ISSM, from the [initMIP Antarctica](https://theghub.org/groups/ismip6/wiki/initMIP-Antarctica),

See below a very crude snapshot from a holoviews animation I did not long ago... I've zoom into the glaciers that @B-Miles mentioned above 
![image](https://github.com/IACS-cryo/Delineation-WG/assets/16323300/1d548d72-27f4-4079-b53f-00a83b60e056)

Plotting the data on a map is quite tricky because the NetCDF files from Ghub lack projection information. I assume this is re-gridded output from the ice sheet model's initial surface velocities, which are then used for ISMIP projections. However, the issue may be that these glaciers not only appear disconnected from the ice sheet, as Bertie mentioned, but in my opinion, they can't be effectively modeled with a large-scale ice sheet model. Perhaps a higher resolution version of ISSM, focused on that specific region ? Wonder how the actual output without re-gridding looks like? 

I believe we need a broader discussion on establishing a connectivity level between the Peripheral Glaciers and the Ice Sheet. This discussion should also link or correlate this connectivity with model resolution (in both modelling communities).

**Reply by MathieuMorlighem** [reply]
Hello @bearecinos and @B-Miles 

All model output from ISMIP6 are projected on a regular grid (typically 8 km). Here is the model velocity on our native mesh:
![Screenshot 2024-07-04 at 3 50 59 PM](https://github.com/IACS-cryo/Delineation-WG/assets/43073461/3e244c70-b0d3-4bc3-9a9b-52e5d07dde3b)
Keep in mind that this model is more than 7 years old and had ~400,000 elements. We can do better today 😃

Personally, I still think that it is not a good idea to decide whether an ice body is a glacier or ice sheet based on what the best tool is today (I don't think I have seen a study that showed that one tool would be "better" than the other). The problem is that connectivity is very difficult to define here because it is continuously covered in ice... 
![Screenshot 2024-07-04 at 4 07 36 PM](https://github.com/IACS-cryo/Delineation-WG/assets/43073461/ee616e3d-d2f7-4f07-a6f8-10ab0652cd48)

I was also talking to Hélène and something that would be very hard for models to do is to dynamically decide whether an ice body is a glacier or an ice sheet. For example, as outlet glaciers retreat, ice masses disconnected from the main ice sheet may form. If we subtract these volumes of ice from the ice sheet total mass, we would see that the ice sheet is losing a lot of mass, but periphery glaciers may artificially "gain mass" because we are shifting where any given pixel should be accounted for. Something a lot easier would be to have a static mask, e.g., defined using today's delineation, and use this to account for future mass change.

I hope this makes sense, and thanks for this interesting discussion
Mathieu

**Reply by bearecinos** [reply]
@MathieuMorlighem yes that makes total sense! thanks for the clarification and the nice model velocities at higher resolution, a community agreement on a static mask and delineation will be easier indeed :)
